<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite skipped="0" hostname="UTISLAPTOP50" name="StructureComparison.DataComparison.MainFile" tests="1" failures="0" timestamp="18 Jun 2020 09:59:59 GMT" time="8.608" errors="0">
  <testcase name="readsoucefromCSV" time="8.608" classname="StructureComparison.DataComparison.MainFile"/>
=======
<testsuite skipped="0" hostname="UTISLAPTOP54" name="StructureComparison.DataComparison.MainFile" tests="3" failures="0" timestamp="17 Jun 2020 15:21:41 GMT" time="29.310" errors="1">
  <testcase name="readsourcefromdb" time="20.562" classname="StructureComparison.DataComparison.MainFile">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at StructureComparison.DataComparison.DataCompare.compareTwoSheets(DataCompare.java:69)
at StructureComparison.DataComparison.MainFile.readsourcefromdb(MainFile.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- readsourcefromdb -->
  <testcase name="readsourcefromexcel" time="7.957" classname="StructureComparison.DataComparison.MainFile"/>
  <testcase name="readsoucefromcsv" time="0.791" classname="StructureComparison.DataComparison.MainFile"/>
>>>>>>> 21f5d76f6f837781b28e1ec369ade4e2276dada8
</testsuite> <!-- StructureComparison.DataComparison.MainFile -->
